### ISSUE TITLE
[Julia Package] cannot find lib_lightgbm in system dirs

### ISSUE THREAD

**Issue by nesteiner** [microsoft/LightGBM#5229]
![图片](https://user-images.githubusercontent.com/46296608/169690295-a2196a57-cb0a-4835-8981-4b9918c8417f.png)
 
I have compile LightGBM from source code, and link the /usr/local/lib/lib_lightgbm.so to /usr/lib/lib_lightgbm.so,  but can't still find lib_lightgbm in julia repl, what is going on ??

**Comment by StrikerRUS** [comment]
Hey @nesteiner !
When you compile LightGBM from source on Linux, `lib_lightgbm.so` file will be located in the root folder of the git repo: `LightGBM/lib_lightgbm.so`.

**Comment by nesteiner** [comment]
yes, I know that. and then what should I do, may I ask ? :slightly_smiling_face:

**Comment by jameslamb** [comment]
Sorry @nesteiner , we do not maintain `LightGBM.jl` here in this repo. If the issue you're facing is specific to "`LightGBM.jl` cannot find `lib_lightgbm.so`", please open an issue at https://github.com/IQVIA-ML/LightGBM.jl/issues.

**Comment by StrikerRUS** [comment]
@nesteiner 
It depends on what you want to get.

I guess you need to do this: https://github.com/IQVIA-ML/LightGBM.jl#custom-lightgbm-binaries.

**Comment by nesteiner** [comment]
ok, I see that

**Comment by yaxxie** [comment]
Hello @nesteiner if you're continuing to have trouble, please open an issue https://github.com/IQVIA-ML/LightGBM.jl/issues

I think this one can be closed @StrikerRUS @jameslamb
